portfolio/src/main/webapp/index.html

```diff
@@ -7,28 +7,42 @@
     <script src="script.js"></script>
   </head>
   <body onload="loadData()">
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is limli's portfolio.</p>
-      <p>
-          I like playing puzzle games, and my favourite is Stephen's Sausage Roll, because the game is very <i>elegant</i>.
-      </p>
-      <p><img src="/images/ssr.jpg" width="400em"></p>
-      <p><a href="https://github.com/limli/">My Github</a></p>
-      <p>Click the button below to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-      <div>
-        <h3>Comments</h3>
-        <div id="comments-container"></div>
+    <div class="content">
+      <div class="header">
+        <h2>Lim Li's Portfolio</h2>
       </div>
-      <div>
-        <h3>Leave a Comment</h3>
-        <form action="/data" method="POST">
-          <textarea class="comment-textarea" name="comment" placeholder="Your Comment..."></textarea>
-          <br>
-          <input type="submit" value="Add a Comment"/>
-        </form>
+
+      <div class="row">
+        <div class="leftcolumn">
+          <div class="card">
+            <h2>About Me</h2>
+            <p>
+              I am a year 2 NUS student studying Computer Science and Applied Math.
+            </p>
+            <p>
+              I like playing puzzle games, and my favourite is Stephen's Sausage Roll, because the game is very <i>elegant</i>.
+            </p>
+            <p><img src="/images/ssr.jpg" width="300em"></p>
```

Comment:
Put alternative text for images.

ref - https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback